internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -103,31 +121,42 @@ func resourceDataFactoryLinkedServiceSFTP() *pluginsdk.Resource {
 				},
 			},
 
-			"skip_host_key_validation": {
-				Type:     pluginsdk.TypeBool,
-				Optional: true,
+			"passphrase": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Specify the pass phrase or password to decrypt the private key if the key file or the key content is protected by a pass phrase",
+				ConflictsWith: []string{"password"},
 			},
 
-			"host_key_fingerprint": {
-				Type:         pluginsdk.TypeString,
-				Optional:     true,
-				ValidateFunc: validation.StringIsNotEmpty,
+			"password": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Sensitive:     true,
+				ValidateFunc:  validation.StringIsNotEmpty,
+				ConflictsWith: []string{"private_key_content", "private_key_path", "passphrase"},
+				AtLeastOneOf:  []string{"private_key_content", "private_key_path"},
```

Comment:
Though I do believe this work fine as the argument where `AtLeastOneOf` is defined is included by default, I wonder if it makes sense to be explicit about it like the arguments below? 
```suggestion
				AtLeastOneOf:  []string{"password", "private_key_content", "private_key_path"},
```

Reply:
certainly